AATAMS/AATAMS_acoustic/check_csv.py

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env python
+"""
+Simple check (number of columns) of csv file format
```

Comment:
Do you just want to check the number of columns, or should the column headers also match the required names?